extensions/sql-database-projects/src/models/project.ts

```diff
@@ -347,13 +347,10 @@ export class Project {
 	public async deleteFileFolder(entry: FileProjectEntry): Promise<void> {
 		// compile a list of folder contents to delete; if entry is a file, contents will contain only itself
 		const toDeleteFiles: FileProjectEntry[] = this.files.concat(this.preDeployScripts).concat(this.postDeployScripts).concat(this.noneDeployScripts).filter(x => x.fsUri.fsPath.startsWith(entry.fsUri.fsPath) && x.type === EntryType.File);
-		const toDeleteFolders: FileProjectEntry[] = this.files.filter(x => x.fsUri.fsPath.startsWith(entry.fsUri.fsPath) && x.type === EntryType.Folder).sort(x => -x.relativePath.length);
+		const toDeleteFolders: FileProjectEntry[] = this.files.filter(x => x.fsUri.fsPath.startsWith(entry.fsUri.fsPath) && x.type === EntryType.Folder);
 
 		await Promise.all(toDeleteFiles.map(x => fs.unlink(x.fsUri.fsPath)));
-
-		for (const folder of toDeleteFolders) {
-			await fs.rmdir(folder.fsUri.fsPath); // TODO: replace .sort() and iteration with rmdir recursive flag once that's unbugged
```

Comment:
@Benjin I tested the recursive flag and it appears to work, but I can't find the PR that fixed it/documentation where it comes from. Do you know where I can find that?

Reply:
Hmm, I'm not seeing it either.  https://github.com/cjihrig/node/commits/master/lib/internal/fs/rimraf.js

If it's working and tests are passing on all platforms, then I'd say it's good to check in.